internal/contentcache/contentcache.go

```diff
@@ -36,6 +40,34 @@ type ContentCache struct {
 	mtimeClock     timeutil.Clock
 }
 
+// Populates the cache with existing persisted files when gcsfuse starts
+func (c *ContentCache) PopulateCache() {
```

Comment:
PopulateCache sounds like adding stuff to an empty cache. Would "RecoverCache" or "SetupCache" be a better name?

Reply:
RecoverCache makes sense.